litecov.cpp

```diff
@@ -305,7 +307,7 @@ void LiteCov::ClearCoverage() {
 
 // fetches and decodes coverage from the remote buffer
 void LiteCov::CollectCoverage(ModuleCovData *data) {
-  if (!IsTargetAlive() || !data->has_remote_coverage) return;
+  if (!data->has_remote_coverage) return;

```

Comment:
Removing IsTargetAlive check here is OK, but you should also do it in LiteCov::OnProcessExit